.github/workflows/release-step-3.yml

```
@@ -213,17 +213,29 @@ jobs:
     runs-on: ubuntu-latest
     needs:
       - post-release
-    permissions:
-      contents: write
-    env:
-      GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
     steps:
+      - name: Get token
+        id: get_token
+        uses: tibdex/github-app-token@3beb63f4bd073e61482598c45c71c1019b59b73a # v2.1.0
+        with:
+          app_id: ${{ secrets.OBS_AUTOMATION_APP_ID }}
+          private_key: ${{ secrets.OBS_AUTOMATION_APP_PEM }}
+          permissions: >-
+            {
+              "contents": "write",
+              "pull_requests": "write"
+            }
+          repositories: >-
+            ["${{github.repository}}"]
```

See https://github.com/tibdex/github-app-token/issues/104 or https://github.com/tibdex/github-app-token/issues/106
```suggestion
            ["apm-agent-java"]
```